app/controllers/books_controller.rb

```diff
@@ -6,4 +6,11 @@ def index
   def show
     render json: Book.find(params[:id])
   end
+
+  def external_api_get
```

Comment:
This is not a good method name:
* Try to avoid naming methods with _get_ or _set_
* What would happend if the app have another external_api? This method is not very descriptive of the external api to ask